src/main/java/com/arpnetworking/clusteraggregator/client/AggClientConnection.java

```diff
@@ -283,9 +283,18 @@ private void processMessages() {
             AggClientConnection.class,
             Duration.ofSeconds(30));
     private static final boolean IS_ENABLED;
-    private static final String HOST_KEY = "host";
-    private static final String SERVICE_KEY = "service";
-    private static final String CLUSTER_KEY = "cluster";
+    /**
+     * The key used in dimensions to specify the host-dimension.
+     */
+    public static final String HOST_KEY = "host";
```

Comment:
Yeah there's no good spot for them right now. `MetricData` I guess is better than here though. Will do.